.travis.yml

```diff
@@ -209,12 +241,20 @@ script:
         USE_WHEEL_BUILD="--no-build"
     fi
   - export SCIPY_AVAILABLE_MEM=3G
-  - python -u runtests.py -g -m $TESTMODE $COVERAGE $USE_WHEEL_BUILD -- -rfEX -n 3 2>&1 | tee runtests.log
+  - |
+    if [[ "$TRAVIS_CPU_ARCH" == "arm64" ]]; then
```

Comment:
Why not running 4 threads on all architectures?

Reply:
Does Travis have 4 cores on all archs?

Reply:
travisCI has 2 cores on all archs. https://docs.travis-ci.com/user/reference/overview/

Reply:
We now use `-n 3`. I assume we tested that that was faster than `-n 2` at some point. We should probably do the same here for arm64 as for x86, no need to deviate.

Reply:
This comment isn't addressed yet - the whole `if-else` seems unnecessary. This `runtests.py` command doesn't need touching at all.

Reply:
pushed a change